### ISSUE TITLE
Agamand Heirlooms Quest Bug

### ISSUE THREAD

**Issue by alex1156** [dalaranwow/dalaran-wow#5691]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The weapon stand that should hold the Agamand Family Axe, (from the quest Agamand Heirlooms) on the second basement floor of the Crypt of Agamand Mills, is empty. This makes the quest unable to be completed.
![wowscrnshot_021619_224742](https://user-images.githubusercontent.com/31416213/52908538-6deaa980-323d-11e9-8be8-0fe604193d6a.jpg)

**Expected behaviour**: There should be the quest item called Agamand Family Axe within the weapon stand stated above.

**Steps to reproduce the problem**:

1. Complete the previous Warrior Quest chains: "Speak with Dilinger" from Austil de Mon or Baltus Fowler, "Ulag the Cleaver" from Deathguard Dilinger, and "Speak with Coleman" from Deathguard Dilinger
2. After accepting the Warrior Quest "Agamand Heirlooms" from Coleman Farthing, travel to the Agamand Family Crypt at the Northeastern-most part of Agamand Mills, located in Tirisfal Glades.
3. On the second basement floor, in the room on the right, there is a weapon stand labeled "Agamand Weapon Wrack" which is supposed to hold the Agamand Family Axe. If you manually loot it, you will find the container to be empty.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alex1156** [comment]
Bug has been fixed, thanks to whomever fixed it.

**Comment by Golganeth** [comment]
This was fixed in-game. It's a rare and general issue with game objects. The quest itself works fine.
Closing
